tmt/utils.py

```diff
@@ -127,8 +127,9 @@ def unrooted(self) -> 'Path':
 
 # Maximum number of lines of stdout/stderr to show upon errors
 OUTPUT_LINES = 100
-# Default output width
-OUTPUT_WIDTH = 79
+# Default & actual output width
+DEFAULT_OUTPUT_WIDTH = 79
+OUTPUT_WIDTH = int(os.getenv('TMT_TERM_WIDTH', DEFAULT_OUTPUT_WIDTH)) or DEFAULT_OUTPUT_WIDTH
```

Comment:
Could we rather sanity check if somebody does not provide here some crap like `TMT_TERM_WIDTH` is a string not converatble to string?

Reply:
Added in 7efe03da